src/main/java/backend/services/mail/MailServiceImpl.kt

```diff
@@ -20,6 +20,26 @@ class MailServiceImpl(configurationService: ConfigurationService,
                       private val mailSenderService: MailSenderService) : MailService {
 
     private val host: String = configurationService.getRequired("org.breakout.api.host")
+    private val year = 2019
+
+    private val paymentDeadlineMonthGerman = "Mai"
+    private val paymentDeadlineMonthEnglish = "May"
+    private val paymentDeadlineDay = 22
+
+    private val eventMonthGerman = "Juni"
+    private val eventMonthEnglish= "June"
+    private val eventDay = 14
+
+    private val ceremonyMonthGerman = "Juni"
+    private val ceremonyMonthEnglish= "June"
+    private val ceremonyDay = 14
```

Comment:
The actual ceremony date isn't set yet. Soo leaving this dummy for now, since these emails won't be sent yet